client/src/components/ConcludedIssue.js

```diff
@@ -33,6 +33,16 @@ class ConcludedIssue extends React.Component {
     };
   }
 
+  componentWillReceiveProps(nextProps) {
+    this.setState({
+      majority: ConcludedIssue.calculateMajority(
```

Comment:
Sure. As you mentioned, that isn't related to this PR, so I'll create an issue for it instead.